presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -144,6 +180,51 @@ public void testCreatePartitioned()
         }
     }
 
+    @Test
+    public void testCreateExternalPartitioned()
+    {
+        String tableName = "test_basic_statistics_external_partitioned_presto";
+        String helperTableName = tableName + "_helper";
+
+        onPresto().executeQuery(format("DROP TABLE IF EXISTS %s", helperTableName));
+        onPresto().executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        onPresto().executeQuery(format("" +
+                "CREATE TABLE %s " +
+                "WITH (" +
+                "   partitioned_by = ARRAY['n_regionkey'] " +
+                ") " +
+                "AS " +
+                "SELECT n_nationkey, n_name, n_comment, n_regionkey " +
+                "FROM nation " +
+                // turns out there are exactly 5 countries in each region
+                // let's change records count for one of the regions to verify statistics are different
+                "WHERE n_nationkey <> 23", helperTableName));
+
+        try {
+            Optional<String> location = getTableLocation(helperTableName);
+            if (location.isPresent()) {
+                onPresto().executeQuery(format("" +
+                        "CREATE TABLE %s (" +
+                        "   n_nationkey bigint, " +
+                        "   n_name varchar(25), " +
+                        "   n_comment varchar(152), " +
+                        "   n_regionkey bigint) " +
+                        "WITH (partitioned_by = ARRAY['n_regionkey'], external_location = '%s')",
+                        tableName,
+                        location.get()));
+            }
+            else {
+                fail("Could not determine location of managed table");
+            }
+            BasicStatistics statistics = getBasicStatisticsForTable(onHive(), tableName);
+            assertThatStatisticsAreZero(statistics);
```

Comment:
Maybe in partitioned table case, we could call sync_partitions procedure and verify again.
(Unless it wouldn't work in that case, then keep as it is.)